jetty-core/jetty-http3/jetty-http3-server/src/main/java/org/eclipse/jetty/http3/server/HTTP3ServerConnectionFactory.java

```diff
@@ -102,44 +99,33 @@ public void onFailure(Session session, long error, String reason, Throwable fail
 
     private static class HTTP3StreamListener implements Stream.Server.Listener, Invocable
     {
-        private final HTTP3Stream stream;
+        private HTTP3Stream http3Stream;
```

Comment:
I would rather directly store the `ServerHTTP3StreamConnection` and `InvocationType` instances.